ml4h/recipes.py

```diff
@@ -629,6 +711,53 @@ def _get_predictions(args, models_inputs_outputs, input_data, outputs, input_pre
     return predictions
 
 
+def _get_segmentation_evaluations(args, models_inputs_outputs, input_data, outputs, input_prefix, output_prefix):
+    """Makes multi-modal predictions for a given number of models.
+
+    Returns:
+        dict: The nested dictionary of predicted values.
+
+            {
+                'tensor_map_1':
+                    {
+                        'model_1': [[value1, value2], [...]],
+                        'model_2': [[value3, value4], [...]]
+                    },
+                'tensor_map_2':
+                    {
+                        'model_2': [[value5, value6], [...]],
+                        'model_3': [[value7, value8], [...]]
+                    }
+            }
+    """
+    # Local includes only required in this case.
+    import ml4h.evaluate_segmentations as evaluate
```

Comment:
For consistency, use file level import and prefer explicit imports without aliasing when possible, so `from ml4h.evaluate_segmentations import evaluate_segmentation_models`